OJS.Workers.ExecutionStrategies/RubyExecutionStrategy.cs

```diff
@@ -9,8 +9,12 @@
 
     public class RubyExecutionStrategy : ExecutionStrategy
     {
-        public RubyExecutionStrategy(string rubyPath, int baseTimeUsed, int baseMemoryUsed)
-            : base(baseTimeUsed, baseMemoryUsed) =>
+        public RubyExecutionStrategy(
+            IProcessExecutorFactory processExecutorFactory,
+            string rubyPath,
+            int baseTimeUsed,
+            int baseMemoryUsed)
+            : base(processExecutorFactory, baseTimeUsed, baseMemoryUsed) =>
```

Comment:
Is this project following the "arrow on new line convention". I think I saw it on new line above.